.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "liveServer.settings.port": 5501
```

Comment:
Normaal gesproken laten we wijzigingen aan editor settings buiten een feature branch. Hoewel we het in dit geval wel zo kunnen laten (als onderdeel van deze PR)